# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-lt-build"

pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.69; am-1.16; lt-2.4.6)
libtoolize -c -f; aclocal; autoheader; autoconf; automake -a -c
./configure --disable-static --prefix=

# build
make

# install
install -d $DESTDIR/include/pixman-1
install -m 644 pixman/pixman.h $DESTDIR/include/pixman-1
install -m 644 pixman/pixman-version.h $DESTDIR/include/pixman-1
install -d $DESTDIR/lib
install pixman/.libs/libpixman-1.so.0 $DESTDIR/lib
ln -sf libpixman-1.so.0 $DESTDIR/lib/libpixman-1.so
install -d $DESTDIR/lib/pkgconfig
install -m 644 pixman-1.pc $DESTDIR/lib/pkgconfig

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/pixman-build.pc << EOF
Name: pixman
Version: 0.36.0
Description: pixel manipulation library
Requires: $REQUIRES

devel=\\
/include/pixman-1/pixman.h \\
/include/pixman-1/pixman-version.h \\
/lib/libpixman-1.so \\
/lib/pkgconfig/pixman-1.pc

exec=\\
/lib/libpixman-1.so.0
EOF

exit

# patch
--- pixman-0.36.0.orig/configure.ac
+++ pixman-0.36.0/configure.ac
@@ -59,11 +59,14 @@
 m4_define([pixman_version],[pixman_major.pixman_minor.pixman_micro])
 
 AC_INIT(pixman, pixman_version, [pixman@lists.freedesktop.org], pixman)
+AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([foreign dist-bzip2])
 
 # Suppress verbose compile lines
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 
+AM_MAINTAINER_MODE
+
 AC_CONFIG_HEADERS(config.h)
 
 AC_CANONICAL_HOST
--- pixman-0.36.0.orig/demos/Makefile.am
+++ pixman-0.36.0/demos/Makefile.am
@@ -1,3 +1,5 @@
+AUTOMAKE_OPTIONS = subdir-objects
+
 EXTRA_DIST = parrot.c parrot.jpg scale.ui
 
 if HAVE_GTK

# source
https://cairographics.org/releases/pixman-0.36.0.tar.gz
